<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ActiveTraining</class>
 <widget class="QWidget" name="ActiveTraining">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1700</width>
    <height>944</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>1188</x>
     <y>30</y>
     <width>488</width>
     <height>784</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#groupBox{background: #FFFFFF;
border-radius: 12px;}</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QGroupBox" name="train_GroupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>448</width>
      <height>196</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>17</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#train_GroupBox{border-radius: 8px;
border: 1px solid #979797;
margin-top:4ex;}
#train_GroupBox::title{subcontrol-origin:margin;
subcontrol-position:top center;
padding:0 5px;}</string>
    </property>
    <property name="title">
     <string>训练设置</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <widget class="QSpinBox" name="timeLimit_SpinBox">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>119</y>
       <width>71</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="value">
      <number>20</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>202</x>
       <y>125</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>秒</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>125</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>时间限制</string>
     </property>
    </widget>
    <widget class="QPushButton" name="controlSet_Btn">
     <property name="geometry">
      <rect>
       <x>69</x>
       <y>40</y>
       <width>300</width>
       <height>44</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>19</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{background: #0D9DDB;border-radius: 4px;color:white;}
QPushButton:hover{background: #1EAACF;border-radius: 4px;color:#333333;}</string>
     </property>
     <property name="text">
      <string>控制设置</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>125</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>显示</string>
     </property>
    </widget>
    <widget class="QPushButton" name="display_Btn">
     <property name="geometry">
      <rect>
       <x>338</x>
       <y>127</y>
       <width>44</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/new/prefix1/source/buttons/generalButton/turnOff.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="moveType_GroupBox">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>448</width>
      <height>220</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>17</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#moveType_GroupBox{border-radius: 8px;
border: 1px solid #979797;
margin-top:4ex;}
#moveType_GroupBox::title{subcontrol-origin:margin;
subcontrol-position:top center;
padding:0 5px;}</string>
    </property>
    <property name="title">
     <string>活动类型</string>
    </property>
    <widget class="QPushButton" name="bucklingType_Btn">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>41</y>
       <width>60</width>
       <height>60</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{border-image: url(:/new/prefix1/source/buttons/activeTraining/bucklingTypeNormal.png);}
QPushButton:hover{border-image: url(:/new/prefix1/source/buttons/activeTraining/bucklingTypeHover.png);}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="stretchType_Btn">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>41</y>
       <width>60</width>
       <height>60</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{border-image: url(:/new/prefix1/source/buttons/activeTraining/stretchTypeNormal.png);}
QPushButton:hover{border-image: url(:/new/prefix1/source/buttons/activeTraining/stretchTypeHover.png);}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="randomType_Btn">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>41</y>
       <width>60</width>
       <height>60</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton:hover{border-image: url(:/new/prefix1/source/buttons/activeTraining/randomTypeHover.png);}
QPushButton{border-image: url(:/new/prefix1/source/buttons/activeTraining/randomTypeNormal.png);}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="autoType_Btn">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>131</y>
       <width>60</width>
       <height>60</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{border-image: url(:/new/prefix1/source/buttons/activeTraining/autoTypeNormal.png);}
QPushButton:hover{border-image: url(:/new/prefix1/source/buttons/activeTraining/autoTypeHover.png);}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="manualType_Btn">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>131</y>
       <width>60</width>
       <height>60</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{border-image: url(:/new/prefix1/source/buttons/activeTraining/manualTypeNormal.png);}
QPushButton:hover{border-image: url(:/new/prefix1/source/buttons/activeTraining/manualTypeHover.png);}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_19">
     <property name="geometry">
      <rect>
       <x>29</x>
       <y>44</y>
       <width>111</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>方向选择</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_20">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>133</y>
       <width>111</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>手指选择</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_21">
     <property name="geometry">
      <rect>
       <x>159</x>
       <y>197</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>手动</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_22">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>197</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>自动</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_23">
     <property name="geometry">
      <rect>
       <x>341</x>
       <y>105</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>随机</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_24">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>105</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>屈曲</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_25">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>105</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>伸展</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="timeInterval_GroupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>560</y>
      <width>448</width>
      <height>130</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>17</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#timeInterval_GroupBox{border-radius: 8px;
border: 1px solid #979797;
margin-top:4ex;}
#timeInterval_GroupBox::title{subcontrol-origin:margin;
subcontrol-position:top center;
padding:0 5px;}</string>
    </property>
    <property name="title">
     <string>训练难易程度</string>
    </property>
    <widget class="QPushButton" name="level3_Btn">
     <property name="geometry">
      <rect>
       <x>284</x>
       <y>40</y>
       <width>60</width>
       <height>60</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{border-image: url(:/new/prefix1/source/buttons/level/level3NoSelect.png);}
QPushButton:hover{border-image: url(:/new/prefix1/source/buttons/level/level3Hover.png);}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="level1_Btn">
     <property name="geometry">
      <rect>
       <x>104</x>
       <y>40</y>
       <width>60</width>
       <height>60</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{border-image: url(:/new/prefix1/source/buttons/level/level1NoSelect.png);}
QPushButton:hover{border-image: url(:/new/prefix1/source/buttons/level/level1Hover.png);}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="level2_Btn">
     <property name="geometry">
      <rect>
       <x>194</x>
       <y>40</y>
       <width>60</width>
       <height>60</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{border-image: url(:/new/prefix1/source/buttons/level/level2NoSelect.png);}
QPushButton:hover{border-image: url(:/new/prefix1/source/buttons/level/level2Hover.png);}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="finish_Btn">
   <property name="geometry">
    <rect>
     <x>1489</x>
     <y>845</y>
     <width>164</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{background: #0D9DDB;border-radius: 4px;color:white;}
QPushButton:hover{background: #1EAACF;border-radius: 4px;color:#333333;}</string>
   </property>
   <property name="text">
    <string>完成</string>
   </property>
  </widget>
  <widget class="QPushButton" name="start_Btn">
   <property name="geometry">
    <rect>
     <x>1188</x>
     <y>845</y>
     <width>164</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{background: #0D9DDB;border-radius: 4px;color:white;}
QPushButton:hover{background: #1EAACF;border-radius: 4px;color:#333333;}</string>
   </property>
   <property name="text">
    <string>开始</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="score_ProgressBar">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>174</y>
     <width>460</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#score_ProgressBar{border:none;background:#888888; border-radius:10px;}
QProgressBar::chunk 
{
      background-color:#70E086;
      border-radius:10px;
}</string>
   </property>
   <property name="value">
    <number>100</number>
   </property>
   <property name="textVisible">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>234</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #70E086;
border: 1px solid #979797;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>304</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #F79A69;
border: 1px solid #979797;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>374</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #FF767F;
border: 1px solid #979797;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>254</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>正确</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>324</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>超时</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>394</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>错误</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>649</y>
     <width>361</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>超时前请按压/弯曲选择的手指</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="timeOut_ProgressBar">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>710</y>
     <width>360</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#timeOut_ProgressBar{border:none;background:#888888; border-radius:10px;}
QProgressBar::chunk 
{
      background-color:#70E086;
      border-radius:10px;
}</string>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="textVisible">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="CHandStateShow" name="hand_Widget" native="true">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>173</y>
     <width>560</width>
     <height>560</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>656</x>
     <y>200</y>
     <width>458</width>
     <height>18</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/source/dividingRule/horizontal.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>199</x>
     <y>70</y>
     <width>271</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#333333;</string>
   </property>
   <property name="text">
    <string>主动运动</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="trainTips_Label">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>70</y>
     <width>241</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>准备中...</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CHandStateShow</class>
   <extends>QWidget</extends>
   <header>chandstateshow.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
